src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -2,12 +2,17 @@
 
   <PropertyGroup>
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
-
-    <!-- NuGet -->
     <Version>1.10.0</Version>
     <Title>Datadog APM - ClrProfiler</Title>
-    <Description>Automatic instrumentation library used by .NET Tracer for Datadog APM. Requires separate native library that implements the CLR Profiling API.</Description>
+    <Description>OBSOLETE. This package is deprecated and exists only for backwards compatibility.
+      Users who need manual instrumentation should reference the "Datadog.Trace" package.
+      Users who need ASP.NET instrumentation should reference the "Datadog.Trace.AspNet" package.
+   </Description>	
     <Authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</Authors>
+
+    <!-- Remove the Datadog.Trace.ClrProfiler.Managed.dll assembly from the output and hide inapplicable warning (https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128) -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
```

Comment:
Aaaaand I just realized that you'll need to add the ProjectReference to Datadog.Trace.AspNet in this project.

Reply:
This is what I get for accepting you changes blindly, haha.

Reply:
Fixed in 853b811985bc5b95692c20a646a9cefb71db74cb.